event-handler/storage/localhost_3025_start_time

```diff
@@ -1 +0,0 @@
-2020-01-01T00:00:00Z
```

Comment:
Added by mistake?

Reply:
Yes, debug artefact, removed by this PR.